test/parallel/test-cluster-setup-master.js

```diff
@@ -49,7 +49,7 @@ if (cluster.isWorker) {
   cluster.once('setup', function() {
     checks.setupEvent = true;
 
-    const settings = cluster.settings;
+    settings = cluster.settings;
```

Comment:
Is this change actually needed at all?

Reply:
it's to be able to stringify the "settings" variable inside the assert message.

Reply:
@BridgeAR Hey, just wanna be sure about "This effects the test subsystem and the commit message do not follow the guidelines that are linked when opening a PR. It would be nice if this could be fixed." you want me to rewrite each commit message to make sure they follow the guide line ?!

Reply:
@jbeatj If you are comfortable with git, you can do a `git rebase master -i` and squash all the commits with one left as something like `test: improve the assertion message`

Reply:
@jbeatj Also you can leave this to whoever lands this and they would do that for you.